leasing/models/rent.py

```diff
@@ -1476,6 +1476,118 @@ class LegacyIndex(models.Model):
     )
 
 
+class OldDwellingsInHousingCompaniesPriceIndex(models.Model):
+    """
+    In Finnish: Vanhojen osakeasuntojen hintaindeksi
+
+    From my understanding, a unique price index is identified by the API
+    database table URL and code of the index. A single table can hold multiple
+    indexes, and a single code can be used in multiple tables.
+
+    Members:
+        code: Code for the index's table column. Example: "ketj_P_QA_T". Same code for different intervals, e.g. yearly or quarterly.
+        name: Name of the index. Example: "Index (2020=100)".
+        comment: Comment for the index's table column.
+        source: Source of the data.
+        source_table_updated: UTC timestamp when the source table was last updated.
+        source_table_label: Label for the source table.
+        url: API endpoint URL.
+    """
+
+    # Maximum lengths are arbitrary, but set to avoid extra large input.
+    CHARFIELD_MAX_LENGTH = 255
+
+    code = models.CharField(
+        verbose_name=_("Code"),
+        max_length=CHARFIELD_MAX_LENGTH,
+    )
+    name = models.CharField(
+        verbose_name=_("Name"),
+        max_length=CHARFIELD_MAX_LENGTH,
+    )
+    comment = models.TextField(verbose_name=_("Region"), blank=True)
```

Comment:
The existing convention in DB models in backend seems to be to translate verbose name of each field. 

However, it looks like we go against the Django convention of NOT capitalizing the first letter of verbose name: The convention is not to capitalize the first letter of the verbose_name <https://docs.djangoproject.com/en/4.2/topics/db/models/#verbose-field-names>. Maybe better I align with our existing convention of capitalization instead of making an exception.